hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/MasterSnapshotContext.java

```diff
@@ -111,66 +114,73 @@ void startScheduledSnapshot(long executionId) {
         } finally {
             mc.unlock();
         }
-        tryBeginSnapshot();
+        tryBeginSnapshotPhase1();
```

Comment:
In similar logic we could rename `onSnapshotPhase2Completed` to `onSnapshotCompleted`, because it's the last phase, but I think it's clearer the current way and not too long.